test/libsolidity/syntaxTests/functionTypes/call_value_on_non_payable_function_type.sol

```diff
@@ -1,8 +1,9 @@
 contract C {
     function (uint) external returns (uint) x;
     function f() public {
-        x.value(2)();
```

Comment:
Thinking about it, for a few of these test cases it would be better to not just update the tests to the new syntax, but to keep a copy of the test with the old syntax around for now. This is one of them - I'll leave a comment on each of the other ones.